src/test/java/org/sagebionetworks/bridge/reporter/worker/BridgeReporterProcessorTest.java

```diff
@@ -144,12 +161,12 @@ public void testNormalCase() throws Exception {
     @Test
     public void testNormalCaseWeekly() throws Exception {
         // execute
-        callback.callback(REQUEST_JSON_TEXT_WEEKLY);
+        processor.process(REQUEST_JSON_WEEKLY);
 
         // verify
         verify(mockBridgeHelper).getAllStudiesSummary();
-        verify(mockBridgeHelper).getUploadsForStudy(eq(TEST_STUDY_ID), eq(TEST_START_DATETIME), eq(TEST_END_DATETIME));
-        verify(mockBridgeHelper).saveReportForStudy(eq(TEST_STUDY_ID), eq(TEST_REPORT_ID_WEEKLY), eq(TEST_REPORT));
+        verify(mockBridgeHelper, times(7)).getUploadsForStudy(eq(TEST_STUDY_ID), any(), any());
```

Comment:
We should test that we're calling getUploadsForStudy with the correct parameters. This makes sure we're computing start- and endDateTimes for each iteration of the loop.